Prelude-UIKit/lenses/UIBarButtonItemLenses.swift

```diff
@@ -63,7 +62,10 @@ public extension LensHolder where Object: UIBarButtonItem {
 
   public var targetAction: Lens<Object, (AnyObject, Selector)?> {
     return Lens(
-      view: { item in item.target.map { ($0, item.action) } },
+      view: { item in
+        guard let target = item.target, let action = item.action else { return nil }
+        return (target, action)
```

Comment:
yeah, seems in swift 3 target and selector are now optional on bar button items, whereas previously it was just target.